core/templates/core/profile.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'core/base.html' %}
+
+{% load static %}
+{% load core_extras %}
+
+{% block title %}{{ user.name }}{% endblock %}
+
+{% block content %}
+    <div class="mt-32 flex flex-col items-center">
+        <div class="flex gap-5 md:w-1/2 w-3/4 pb-4 border-b-2">
+            <img src="{% if user.avatar %}{{ user.avatar }}{% else %}{% static "images/placeholder_profile_pic.png" %}{% endif %}" alt="Profile picture" class="lg:w-[12.5%] lg:h-[12.5%] md:w-1/6 md:h-1/6 w-1/4 h-1/4 rounded-full">
```

Comment:
Why you can't change this so width matches height and modify only one property?

Reply:
I did, h-auto, but you had me delete it